sources/platform/runtime/machine-image/language-versions.md

```diff
@@ -307,10 +307,10 @@ The following images are available when you configure `language: python` in your
 	|2.7.12          |- Ubuntu 14.04<br>- Ubuntu 16.04<br>- Ubuntu 16.04(aarch64) <br>- CentOS 7 | [v5.7.3](/platform/runtime/machine-image/ami-v573/) to [v6.5.4](/platform/runtime/machine-image/ami-v654/)<br> [v5.7.3](/platform/runtime/machine-image/ami-v573/) to [v6.5.4](/platform/runtime/machine-image/ami-v654/) <br> [v6.2.4](/platform/runtime/machine-image/ami-v624/) to [v6.5.4](/platform/runtime/machine-image/ami-v654/) <br> [v6.5.4](/platform/runtime/machine-image/ami-v654/)
 	|2.7.6           |- Ubuntu 14.04                    | [v5.6.1](/platform/runtime/machine-image/ami-v561/) and earlier |
 	|2.7.5           |- CentOS 7                        | [v6.2.4](/platform/runtime/machine-image/ami-v624/) to [v6.4.4](/platform/runtime/machine-image/ami-v644/)
-	|2.6.9           |- Ubuntu 14.04<br>- Ubuntu 16.04 <br>- CentOS 7    | [v5.6.1](/platform/runtime/machine-image/ami-v561/) and earlier to [v6.5.1](/platform/runtime/machine-image/ami-v561/)<br> [v6.5.4](/platform/runtime/machine-image/ami-v654/)|
+	|2.6.9           |- Ubuntu 14.04<br>- Ubuntu 16.04 <br>- CentOS 7    | [v5.6.1](/platform/runtime/machine-image/ami-v561/) and [v6.5.4](/platform/runtime/machine-image/ami-v561/) <br>[v5.6.1](/platform/runtime/machine-image/ami-v561/) and [v6.5.4](/platform/runtime/machine-image/ami-v561/)<br> [v6.5.4](/platform/runtime/machine-image/ami-v654/)|
```

Comment:
These links don't make sense. v6.5.4 is listed thrice, and linked to all the wrong places.

Reply:
ok i will change this

Reply:
i have updated the links. v6.5.4 is listed thrice because it corresponds to Ubuntu 14.04, Ubuntu 16.04 and Centos 7 images. Below is the screenshot of how it looks
![v6 5 4](https://user-images.githubusercontent.com/37403043/41039464-24c6d322-69b7-11e8-9b89-cf5d9d789ac3.png)